disco_aws_automation/disco_datapipeline.py

```diff
@@ -174,23 +243,45 @@ def delete(self, pipeline):
             raise DataPipelineStateException("Pipeline must be saved before it can be deleted (but...)")
         self._dp_client.delete_pipeline(pipelineId=pipeline._id)
 
-    def fetch_or_create(self, template_name, pipeline_name, pipeline_description, tags, log_location):
+    def fetch_or_create(self, template_name, pipeline_name, pipeline_description, tags, log_location,
+                        force_update=False, metanetwork=None, availability_zone=None):
```

Comment:
We probably wouldn't, but since I can't actually choose a metanetwork _without_ choosing a specific AZ, it seemed like I ought to expose the ability to choose one explicitly.